src/datumaro/plugins/transforms.py

```diff
@@ -1837,3 +1845,129 @@ def transform_item(self, item: DatasetItem):
         ]
 
         return self.wrap_item(item, annotations=annotations)
+
+
+class Clean(ItemTransform):
+    """ """
+
+    def __init__(
+        self,
+        extractor: IDataset,
+    ):
+        super().__init__(extractor)
+
+        self._outlier_value = {}
+        self._missing_value = {}
+
+    @staticmethod
+    def remove_unneccessary_char(text):
+        if pd.isna(text):
+            return text
+        try:
+            from nltk.corpus import stopwords
+
+            stop_words = set(stopwords.words("english"))  # TODO
+        except LookupError:
+            import nltk
+
+            nltk.download("stopwords")
+            stop_words = set(stopwords.words("english"))  # TODO
+
+        text = re.sub(r"<.*?>", "", text)
+        text = re.sub(r"http\S+|www\S+|https\S+", "", text, flags=re.MULTILINE)
+        text = emoji_pattern.sub(r"", text)
+        text = text.lower()
+        text = re.sub(r"[^A-Za-z\s]+", "", text)
+        text = re.sub(r"\s+", " ", text).strip()
+        text = " ".join([word for word in text.split() if word not in stop_words])
+        return text
```

Comment:
Could you please describe what each line is for in comments?

Reply:
Fix in [cca8b29](https://github.com/openvinotoolkit/datumaro/pull/1520/commits/cca8b29229e1b86bec4745125d20ca3bd03fad61).